README.md

```diff
@@ -96,17 +118,13 @@ For example:
 ```sh
 #!/bin/sh
 
-brew_tap 'caskroom/cask'
-brew_install_or_upgrade 'brew-cask'
+brew_install_or_upgrade 'mongodb'
+brew_launchctl_restart 'mongodb'
 
-brew cask install dropbox
-brew cask install google-chrome
-brew cask install rdio
+brew_cask_install 'alfred'
+brew_cask_install 'dash'
 
-gem_install_or_update 'parity'
```

Comment:
Maybe we should leave this in, as an example of how to install gems with the script?